### ISSUE TITLE
TestDockerReadWriteReferenceVersion in deployment can't find /cockroach

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#6073]
https://circleci.com/gh/cockroachdb/cockroach/16347 failed during the deploy-time run of the acceptance tests (the previous invocation was successful). Note the `/bin/bash: line 20: /cockroach: No such file or directory` below.

```
--- FAIL: TestDockerReadWriteReferenceVersion (6.32s)
+ echo 'Use the reference binary to write a couple rows, then render its output to a file and shut down.'
Use the reference binary to write a couple rows, then render its output to a file and shut down.
+ /reference-version/cockroach sql -e 'CREATE DATABASE old'
CREATE DATABASE
+ /reference-version/cockroach sql -d old -e 'CREATE TABLE testing (i int primary key, b bool, s string unique, d decimal, f float, t timestamp, v interval, index sb (s, b))'
CREATE TABLE
+ /reference-version/cockroach sql -d old -e 'INSERT INTO testing values (1, true, '\''hello'\'', decimal '\''3.14159'\'', 3.14159, NOW(), interval '\''1h'\'')'
INSERT 1
+ /reference-version/cockroach sql -d old -e 'INSERT INTO testing values (2, false, '\''world'\'', decimal '\''0.14159'\'', 0.14159, NOW(), interval '\''234h45m2s234ms'\'')'
INSERT 1
+ /reference-version/cockroach sql -d old -e 'SELECT * FROM testing'
+ /reference-version/cockroach quit
node drained and shutdown: ok

+ wait
initiating graceful shutdown of server
server drained and shutdown completed
+ bin=/cockroach
+ sleep 1
+ /cockroach start
/bin/bash: line 20: /cockroach: No such file or directory
+ echo 'Read data written by referencce version using new binary'
+ /cockroach sql -d old -e 'SELECT * FROM testing'
Read data written by referencce version using new binary
/bin/bash: line 23: /cockroach: No such file or directory
	util_test.go:281: expected success: non-zero exit code: 1
```

**Comment by tschottdorf** [comment]
while you're at it, there's also a typo: `referencce`.

**Comment by dt** [comment]
fixed by #6082
